### PR TITLE
Use correct configuration everywhere

### PR DESCRIPTION
**Pull request by philippmwirth**
# Use correct configuration everywhere

This should solve the issues with the CA certificate.

### CODE REVIEW

File: lightly/api/version_checking.py [hunk 1]
```diff
@@ -52,7 +49,7 @@ def do_version_check(current_version: str):
 
 
 def get_versioning_api() -> VersioningApi:
-    configuration = Configuration()
+    configuration = api_workflow_client.get_api_client_configuration_no_token()
     configuration.host = getenv('LIGHTLY_SERVER_LOCATION', 'https://api.lightly.ai')
```
Comment (japrescott) [inline lightly/api/version_checking.py:53]: this line is not needed anymore

File: lightly/api/api_workflow_client.py [hunk 2]
```diff
@@ -311,4 +311,8 @@ def get_api_client_configuration(
     configuration.ssl_ca_cert = ssl_ca_cert
     configuration.host = host
 
-    return configuration
\ No newline at end of file
+    return configuration
+
+
+def get_api_client_configuration_no_token() -> Configuration:
+    return get_api_client_configuration(token="dummy_token")
```
Comment (japrescott) [inline lightly/api/api_workflow_client.py:318]: call this "no_token_specified" or something

File: lightly/api/version_checking.py [hunk 3]
```diff
@@ -52,7 +49,7 @@ def do_version_check(current_version: str):
 
 
 def get_versioning_api() -> VersioningApi:
-    configuration = Configuration()
+    configuration = api_workflow_client.get_api_client_configuration_no_token()
```
Comment (japrescott) [inline lightly/api/version_checking.py:52]: do we really not have the token here already? We should unless this code is called directly, no? Couldn't we rather pass a flag to not raise an exception in `get_api_client_configuration`?

**Review by japrescott — CHANGES_REQUESTED** [review]

File: lightly/api/version_checking.py (hunk 3 above)
Comment (philippmwirth) [reply lightly/api/version_checking.py:52]: > do we really not have the token here already? 

No, we don't have a token.

> Couldn't we rather pass a flag to not raise an exception in get_api_client_configuration?

We could 👍

**Review by japrescott — APPROVED** [review]

**Pull request merged**